operations/firmware/FAS_Admin_Procedures.md

```diff
@@ -59,14 +64,14 @@ Refer to [Lock and Unlock Management Nodes](../hardware_state_manager/Lock_and_U
 
 ---
 
-<a name="override-an-image-for-an-update"></a>
-
 ## Override an Image for an Update
 
-If an update fails because of `"No Image available"`, it may be caused by FAS unable to
-match the data on the node to find an image in the image list.
+If you wish to update to a firmware image other than the latest image found by FAS, you can override the image for the update.
+This procedure would also be useful if an update fails because of `"No Image available"`, which is caused by FAS unable to match the data on the node to find an image in the image list.
 
-<a name="procedure-1"></A>
+> **WARNING:** Make sure to select the correct image as FAS will force a flash of the device -- using incorrect firmware may make it inoperable.
```

Comment:
```suggestion
> **WARNING:** Make sure to select the correct image because FAS will force a flash of the device -- using incorrect firmware may make it inoperable.
```